src/main/java/com/synopsys/integration/alert/channel/azure/boards/service/AzureBoardsRequestDelegator.java

```diff
@@ -63,6 +63,7 @@ public AzureBoardsRequestDelegator(Gson gson, ProxyManager proxyManager, AzureBo
     public IssueTrackerResponse sendRequests(List<IssueTrackerRequest> requests) throws IntegrationException {
         IssueConfig azureIssueConfig = context.getIssueConfig();
         AzureBoardsProperties azureBoardsProperties = context.getIssueTrackerConfig();
+        azureBoardsProperties.validateProperties();
```

Comment:
Im not sure if this covers the TODO on line 72 for validating the configuration. Was there more to be validated once the connection is established?

Reply:
I think your code handles the TODO. The only other problem I forsee is having the wrong scopes.

Reply:
I guess we might probably want to validate the connection by performing a HEAD request to some endpoint like _projects_.

Reply:
Your code handles the TODO.  Yes if the application is configured with different scopes than we expect then that will cause a failure.

Reply:
Ya I am only validating the user provided configuration, I am not testing the connection to Azure. Is that something we want to add? If it is I will adjust the TODO statement accordingly

Reply:
I think if we start running into issues with scopes not matching up then we will need to make it configurable in the future.  I think for now we can remove the TODO since you handle the user provided fields.